src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -76,54 +83,123 @@ public Shooter(OperatorInterface operatorinterface) {
         shooter.setNeutralMode(NeutralMode.Coast);
         shooter.configSupplyCurrentLimit(shooterLimit);
 
+        ml.createfile("shooterRPMS");
```

Comment:
We do not need to replay this from a file

Reply:
not bein recorded